### PR TITLE
Fix bug with FlxG.mouse.load

### PR DESCRIPTION
**Pull request by NeeEoo**

### CODE REVIEW

File: flixel/input/mouse/FlxMouse.hx [hunk 1]
```diff
@@ -51,7 +51,7 @@ class FlxMouse extends FlxPointer implements IFlxInputManager
 	 * sits at the right "height". Not used on flash with the native cursor API.
 	 */
 	public var cursorContainer(default, null):Sprite;
-	
+
```
Comment (Geokureli) [inline flixel/input/mouse/FlxMouse.hx:54]: for future PRs set your hxformat.json to 
```json
"trailingWhitespace": true
```
and your settings.json should have
```json
"[haxe]": {
		"editor.formatOnSave": true,
		"editor.formatOnSaveMode": "modifications"
	},
	```

Comment (NeeEoo) [reply flixel/input/mouse/FlxMouse.hx:54]: Alright noted!

**Comment by Geokureli** [comment]
always provide example code that demonstrates a way to reproduce the bug, please

**Comment by NeeEoo** [comment]
Nvm im dumb

**Comment by Geokureli** [comment]
> Nvm im dumb

Why? this PR seemed legit to me, @NeeEoo

**Comment by NeeEoo** [comment]
> Why? this PR seemed legit to me, @NeeEoo 

Turns out you were supposed to be to o

```hx
@:bitmap("assets/images/ui/cursor.png")
class CustomCursor extends openfl.display.BitmapData {}

FlxG.mouse.load(new CustomCursor(0,0));
```

Plus my code would have crashed due to it not calling with the 0, 0

**Comment by Geokureli** [comment]
FWIW this function should probably take FlxGraphicAsset or a FlxGraphicSource or something

**Pull request closed without merging**